docs/tutorials/visualization_tutorial.ipynb

```diff
@@ -118,13 +122,13 @@
    ]
   },
   {
-   "metadata": {},
    "cell_type": "code",
-   "outputs": [],
    "execution_count": null,
+   "metadata": {},
+   "outputs": [],
    "source": [
     "# Create initial model instance\n",
-    "model1 = MoneyModel(50, 10, 10)\n",
+    "model = MoneyModel(n=50, width=10, height=10) #keyword arguments\n",
```

Comment:
The read the docs build is failing, this needs to be `model1` or you need to change the `Solaraviz`  (6 lines down) call to `model`

Reply:
made this change .